### ISSUE TITLE
[QUESTION] RRTMG simulation error

### ISSUE THREAD

**Issue by tjansakoo** [geoschem/geos-chem#1246]
I am using RRTMG simulation, when I try to run in fullchem > RRTMG, I found the following 

===============================================================================
GEOS-Chem ERROR: LRAD=T but RRTMG is not defined at compile time!
 -> at Read_Radiation_Menu (in module GeosCore/input_mod.F90)
===============================================================================

===============================================================================
GEOS-Chem ERROR: Error in "Read_Radiation_Menu"!
 -> at Read_Input_File (in module GeosCore/input_mod.F90)
===============================================================================

===============================================================================
GEOS-CHEM ERROR: Error encountered in "Read_Input_File"!
STOP at  -> at GEOS-Chem (in GeosCore/main.F90)
===============================================================================

I found this error first in GEOS chem v12.9.3 so I update it to ver.14 but still found the error. Could you suggest me how to fix this error. 

Best regards,

**Comment by yantosca** [comment]
@tjansakoo: Yes, there were several issues with RRTMG that were not fixed until 13.1.0.  As @lizziel suggested, please use 13.4.0 for now.

**Comment by tjansakoo** [comment]
@yantosca @lizziel Thank you so much for your recommendation.

**Comment by yantosca** [comment]
Thanks @tjansakoo.  I'll close out this issue now.
